api/query/concrete_query.go

```diff
@@ -8,6 +8,13 @@ import (
 	"github.com/web-platform-tests/wpt.fyi/shared"
 )
 
+// AggregationOpts are options for the aggregation format used when collecting
+// the results.
+type AggregationOpts struct {
+	IncludeSubtests bool
```

Comment:
Can you say something about whether these are mutually exclusive or what they do when combined? Is there some documentation that should be updated too?

Reply:
I'm not going to document until we have all of the pieces in place, end-to-end. Don't want to have to change the documentation when we change the impl.